Cargo.toml

```diff
@@ -36,8 +36,9 @@ crossbeam-utils = "0.8.6"
 once_cell = "1.9.0"
 priority-queue = "1.2.1"
 hashbrown = "0.12.0"
-csv = { version = "1.1", optional = true }
+csv = { git = "https://github.com/ryzhyk/rust-csv.git", optional = true }
```

Comment:
Are we reviewing this one too?

Reply:
Sorry, forgot to mention this in the PR.  All I did is exposed the deserializer type from that crate in the public API.  Unfortunately, the crate maintainer isn't keen to accept this change, so we'll have to live with a fork.